metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppetlabs-splunk_hec",
-  "version": "0.5.0",
+  "version": "0.6.0",
```

Comment:
Looking a release after this PR? If so I'd add a quick 0.6.0 section to the changelog while you're at this, then I could go ahead and do a fresh release after merge.